package.json

```diff
@@ -28,7 +28,10 @@
     "koa-bodyparser": "^4.2.1",
     "koa-logger": "^3.2.0",
     "koa-router": "^7.4.0",
-    "nodemailer": "^4.6.7"
+    "mysql2": "^1.5.3",
+    "nodemailer": "^4.6.7",
+    "sequelize": "^4.38.0",
+    "sequelize-cli": "^4.0.0"
```

Comment:
What does the cli in the dependencies list? This goes to the devDependencies, but your not even using it, so no.

Reply:
i used it for migration...